docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -516,3 +516,14 @@ spec:
   query: select * from shimcache
   purpose: Informational
   contributors: puffyCid
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Get running docker containers
+  platforms: macOS, Linux
+  description: Returns the running Docker containers.
+  query: SELECT id, name, image, image_id, state, status FROM docker_containers WHERE state = "running";
+  purpose: Informational
+  contributors: DominusKelvin
+---
```

Comment:
```suggestion
```

I believe you have to remove the trailing `---` to pass the "Deploy Fleet Website" CI test. 

I think this is because the `---` signals the start of a new yaml document. You only need one at the top of the document (before the `apiVersion`)